### DISCUSSION TITLE
Activate DXVK and use DirectX 9 under bottle version

### DISCUSSION THREAD

**Discussion by camillo2008** [cryinkfly/Autodesk-Fusion-360-for-Linux#338] [Q&A]
I am quite new to wine emulation and even newer to bottle emulation, and I have seen that many of you actually use DX9 and DXVK. How can  enable this under bottle emulation? 

The program is fine and smooth while moving the scene, but all the popups like press-pull, move, and others make it all go really slow.
Also, if you have any other suggestions to make it faster, that would be great.

**Comment by sinbadsalmon** [comment]
I am using Intel HD Graphics and had the same issue. I changed the renderer under bottle display settings to GDI and this vastly improved performance to the point where Fusion is now usable - I can be productive using the Linux environment + Fusion. Fusion still gobbles all my RAM for no reason that I can comprehend other than 'just because' which still hurts performance but we can thank Autodesk for that one.

**Reply by bufothefrog** [reply]
Did you do anything special when installing Fusion through bottles? I did a standard install and it doesn't seem to launch for me. No output when I launch with terminal either

**Reply by sinbadsalmon** [reply]
I cannot say I did anything special; install Bottles via Flatpak, install Fusion360 via Bottles installer.

Not directly related but as a 'flextape' fix to the memory issues I was having, I have increased the size of my swap file and this has given me way more breathing space, especially when working with large files. It doesn't prevent all crashes and more RAM is still required but it's given me some performance optimisation in the short term.

**Comment by camillo2008** [comment]
I will try this. In general, the 3D rendering is fine; moving the image around is fast and smooth, but as soon as I render one Windows system popup, it will explode and become really slow, and when I close it, it becomes fast again.

**Reply by camillo2008** [reply]
Those settings seems to improve a lot:
d3d10core = disabled
d3d11 = native
d3d9 = builtin
dxgi = native

while the GDI does make fusion crash

**Reply by sinbadsalmon** [reply]
Good to hear, what GPU and driver set are you using? (for future reference/to help others)
